scripts/build_all.py

```diff
@@ -54,7 +54,7 @@ def _parse_args():
 # Run docker-login so we can pull the cached image
 login_cmd = subprocess.check_output(
     'aws ecr get-login --no-include-email --registry-id {}'.format(args.account).split())
-print('Executing docker login command: '.format(login_cmd))
+print('Executing docker login command: '.format(login_cmd))  # noqa: F523
```

Comment:
F523 is `.format(...) unused positional arguments`

```suggestion
print('Executing docker login command: {}'.format(login_cmd))
```

Reply:
although, on second thought, maybe we shouldn't print that command. doesn't it have a security token in it?

Reply:
Good catch, didn't realize it was missing the braces. It looks like `login_cmd` only includes the account number, unless I'm missing something.